openvm/src/customize_exe.rs

```diff
@@ -369,95 +300,32 @@ fn create_apcs_for_all_blocks<P: IntoOpenVm>(
         .skip(powdr_config.skip_autoprecompiles as usize)
         .take(n_acc)
         .enumerate()
-        .map(|(index, acc_block)| {
+        .map(|(index, block)| {
             tracing::debug!(
                 "Accelerating block of length {} and start idx {}",
-                acc_block.statements.len(),
-                acc_block.start_idx
+                block.statements.len(),
+                block.start_idx
             );
 
             tracing::debug!(
                 "Acc block: {}",
-                acc_block.pretty_print(openvm_instruction_formatter)
+                block.pretty_print(openvm_instruction_formatter)
             );
 
             let apc_opcode = POWDR_OPCODE + index;
 
-            generate_autoprecompile(
-                &acc_block,
-                airs,
-                apc_opcode,
-                bus_map,
+            powdr_autoprecompiles::build(
+                block,
+                vm_config.clone(),
                 powdr_config.degree_bound,
-                &powdr_config.apc_candidates_dir_path,
+                apc_opcode as u32,
+                powdr_config.apc_candidates_dir_path.as_deref(),
             )
             .unwrap()
         })
         .collect()
 }
 
-// OpenVM relevant bus ids:
-// 0: execution bridge -> [pc, timestamp]
-// 1: memory -> [address space, pointer, data, timestamp, 1]
-// 2: pc lookup -> [...]
-// 3: range tuple -> [col, bits]
-// 5: bitwise xor ->
-//    [a, b, 0, 0] byte range checks for a and b
-//    [a, b, c, 1] c = xor(a, b)
-fn generate_autoprecompile<P: IntoOpenVm>(
```

Comment:
This function goes away entirely, because it was just wrapping the apc crate call, but now everything is already in apc crate form from the start.